languages/ru-RU.js

```diff
@@ -0,0 +1,196 @@
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Ваш вариант не соответствует ни одному из возможных: (${possibles})`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time, abortOptions) => `${tag} | **${error}** | Ответьте на это сообщение допустимым значением аргумента в течение **${time}** ${this.pluralize(time, 'секунды', 'секунд', 'секунд')} или отправьте **${abortOptions.join('**, **')}** для отмены.`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPEATING_REPROMPT: (tag, name, time, cancelOptions) => `${tag} | **${name}** является повторяющимся аргументом | Ответьте на это сообщение дополнительными значениями аргумента в течение **${time}** ${this.pluralize(time, 'секунды', 'секунд', 'секунд')} или отправьте **${cancelOptions.join('**, **')}** для отмены.`,
+			MONITOR_COMMAND_HANDLER_ABORTED: 'Отменено',
+			INHIBITOR_COOLDOWN: (remaining) => `Вы только что использовали эту команду. Вы сможете использовать её снова через ${remaining} ${this.pluralize(remaining, 'секунды', 'секунд', 'секунд')}.`,
+			INHIBITOR_DISABLED_GUILD: 'Эта команда была отключена администратором сервера.',
+			INHIBITOR_DISABLED_GLOBAL: 'Эта команда была отключена владельцем бота.',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Недостаточно прав для выполнения команды, отсутствуют следующие разрешения: **${missing}**`,
+			INHIBITOR_NSFW: 'Вы можете использовать NSFW команды только в NSFW каналах.',
+			INHIBITOR_PERMISSIONS: 'У вас недостаточно прав для использования этой команды.',
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `Команда не может быть выполнена, так как на сервере отсутствуют следующие настройки: **${settings.join(', ')}**.`,
+			INHIBITOR_RUNIN: (types) => `Эта команда может быть выполнена только в определённых типах каналов: ${types}.`,
+			INHIBITOR_RUNIN_NONE: (name) => `Команда ${name} не настроена для выполнения ни в одном канале.`,
+			COMMAND_BLACKLIST_DESCRIPTION: 'Добавляет/удаляет пользователей и серверы в чёрный список бота.',
+			COMMAND_BLACKLIST_SUCCESS: (usersAdded, usersRemoved, guildsAdded, guildsRemoved) => [
+				usersAdded.length ? `**Пользователи добавлены**\n${util.codeBlock('', usersAdded.join(', '))}` : '',
+				usersRemoved.length ? `**Пользователи удалены**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**Серверы добавлены**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**Серверы удалены**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: 'Выполняет произвольный JavaScript. Зарезервировано для владельца бота.',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'Команда eval выполняет код внутри себя, любая ошибка будет обработана.',
+				'Так же вы можете использовать флаги --silent, --depth=число и --async для изменения поведения команды.',
```

Comment:
This should be "также" instead of "так же". "Also" is directly translated as "также", whereas "так же" is used for comparing. This is quite tricky, and people often get confused using these words.
```suggestion
				'Также вы можете использовать флаги --silent, --depth=число и --async для изменения поведения команды.',
```